src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        this.setColor(color);
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The `draw()` method should provide more detailed information about the object that is being drawn. For instance, it could include the color and the radius of the circle.